commit cb682ef94c5bee82e1c7ec76e6de09c8157432f4 (refs/changes/60/2860/1)
Author: Rui Loura <rui@joyent.com>
Date:   2017-10-25T10:21:12-04:00 (1 year, 11 months ago)
    
    TOOLS-1904 joyent-gerrit/docs/user/README.md should include details on squash step

diff --git a/docs/user/README.md b/docs/user/README.md
index a0f5313..269c0b8 100644
--- a/docs/user/README.md
+++ b/docs/user/README.md
@@ -196,8 +196,18 @@ change:
      create mode 100644 README.md
     Successfully rebased and updated refs/heads/master.
 
-And then I push that to the magical reference `refs/changes/12` (because this is
-a new PatchSet for Change 12):
+This will open a file that looks something like this:
+
+    pick bb7b7e5 add initial README
+    pick 8f6b671 forgot to add content
+
+Change `pick` to `squash` for the second commit like so:
+
+    pick bb7b7e5 add initial README
+    squash 8f6b671 forgot to add content
+
+Save and exit.  Then I push that to the magical reference `refs/changes/12`
+(because this is a new PatchSet for Change 12):
 
     dap@sharptooth my-playground $ git push cr HEAD:refs/changes/12
     Counting objects: 4, done.
